### DISCUSSION TITLE
头大的问题

### DISCUSSION THREAD

**Discussion by DroganC** [umijs/umi#11589] [Q&A]
页面报错
Unhandled Rejection (TypeError): Cannot read properties of undefined (reading 'call')

启动后页面会报如上错误，把src/loading删了又好了

// src/loading.tsx
import Loading from '@/components/Loading'
export default Loading;

不能这样写吗 ？而且还是偶发性的

**Comment by fz6m** [comment]
详见 https://github.com/umijs/umi/issues/11587#issuecomment-1700323146
